File: Practica_11/src/Java/Fila_B/Anuncio.java
package Java.Fila_B;

public class Anuncio {
	int numero;
	double precio;
	
	public Anuncio(int numerox, double preciox) {
		this.numero=numerox;
		this.precio=preciox;
	}
	public void mostrar() {
		System.out.println("ANUNCIO");
		System.out.println("Nro.: "+this.numero);
		System.out.println("Precio: "+this.precio);
	}
	public int getNumero() {
		return numero;
	}
	public void setNumero(int numero) {
		this.numero = numero;
	}
	public double getPrecio() {
		return precio;
	}
	public void setPrecio(double precio) {
		this.precio = precio;
	}	
}

File: Practica_11/Python/Fila_B/Anuncio.py
class Anuncio:
    def __init__(self, numero, precio):
        self.numero = numero
        self.precio = precio

    def mostrar(self):
        print("ANUNCIO")
        print("Nro.:", self.numero)
        print("Precio:", self.precio)

    def get_numero(self):
        return self.numero

    def set_numero(self, numero):
        self.numero = numero

    def get_precio(self):
        return self.precio

    def set_precio(self, precio):
        self.precio = precio